### ISSUE TITLE
One-liner install command's "Cache-control" doesn't work.

### ISSUE THREAD

**Issue by KEINOS** [KEINOS/Phar_Box3_installer#10]
```
curl -H 'Cache-Control: no-cache' -LSs https://keinos.github.io/Phar_Box3_installer/installer.php | php
```

After some continuous commit, the one-liner-install-command above, which is provided at README, still **lets "GitHub Pages" use the cache** and downloads an old version. Even the GitHub's master branch is up-to-date.

Though, via browser download, it downloads the current one. And also if I add a random value to the download URL, like below, it'll download the current one. 

```
curl -LSs https://keinos.github.io/Phar_Box3_installer/installer.php?12345 | php
```

So, I think it proves that the cache is not working.

- **OS:** macOS HighSierra (OSX 10.13.5)
- **PHP:** PHP 7.2.6 (cli) (built: May 25 2018 06:18:43)
- **cURL:**
    ```
    $ curl --version
    curl 7.54.0 (x86_64-apple-darwin17.0)
    libcurl/7.54.0 LibreSSL/2.0.20 zlib/1.2.11 nghttp2/1.24.0
    Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 
    pop3s rtsp smb smbs smtp smtps telnet tftp 
    Features: AsynchDNS IPv6 Largefile GSS-API Kerberos SPNEGO NTLM 
    NTLM_WB SSL libz HTTP2 UnixSockets HTTPS-proxy
    ```
- **Browser:** Chrome v66.0.3359.181

---

### TL;DR (Current conlution 2018/06/14)

- Better make it simple. <br>Since after some time, the cache will be updated and the installer itself does not update much, so better make it simple. Cache problem is a matter of temporary thing when commits are often made.

**Comment by KEINOS** [comment]
Closed at https://github.com/KEINOS/Phar_Box3_installer/commit/893bc9dc30ce6522cbbd071c890635f3c2d1c0a4 . Pointed to the wrong issue.
